docs-source/content/userguide/managing-domains/accessing-the-domain/admin-console.md

```diff
@@ -1,6 +1,115 @@
+
+   **NOTE**: These instructions assume that you are installing and running the Remote Console Java program externally to your Kubernetes cluster.
+
+1. When you first connect your browser to the Remote Console, which is at `http://localhost:8012` by default, the console will prompt you with a login dialog for a WebLogic Server Administration Server URL. To give the Remote Console access to an Administration Server running in Kubernetes, you can:
+
+   * Use an [Administration Server `NodePort`](#use-an-administration-server-nodeport).
+
+   * Deploy a load balancer with [ingress path routing rules](#configure-ingress-path-routing-rules).
+
+
+#### Use an Administration Server `NodePort`
+
+For the Remote Console to connect to the Kubernetes WebLogic Server Administration Server’s `NodePort`, use the URL:
+
+```
+http://hostname:adminserver-NodePort/
+```
+
+The `adminserver-NodePort` is the port number of the Administration Server outside the Kubernetes cluster.
+For information about the `NodePort` Service on an Administration Server, see the [Domain resource](https://github.com/oracle/weblogic-kubernetes-operator/blob/master/docs/domains/Domain.md) document.
+
+{{% notice warning %}}
+Exposing administrative, RMI, or T3 capable channels using a Kubernetes `NodePort`
+can create an insecure configuration. In general, only HTTP protocols should be made available externally and this exposure
+is usually accomplished by setting up an external load balancer that can access internal (non-`NodePort`) services.
+For more information, see [T3 channels]({{<relref "/security/domain-security/weblogic-channels#weblogic-t3-channels">}}).
+{{% /notice %}}
+
+#### Configure ingress path routing rules
+
+1. Configure an ingress path routing rule. For information about ingresses, see the [Ingress]({{< relref "/userguide/managing-domains/ingress/_index.md" >}}) documentation.
+
+   For an example, see the following `path-routing` YAML file for a Traefik load balancer:
+
+   ```yaml
+   apiVersion: traefik.containo.us/v1alpha1
+   kind: IngressRoute
+   metadata:
+     annotations:
+       kubernetes.io/ingress.class: traefik
+     name: traefik-pathrouting-1
+     namespace: weblogic-domain
+   spec:
+     routes:
+     - kind: Rule
+       match: PathPrefix(`/`)
+       services:
+       - kind: Service
+         name: domain1-adminserver
+         namespace: weblogic-domain
+         port: 7001
+   ```
+
+1. For the Remote Console to connect to the Kubernetes WebLogic Server Administration Server, supply a URL that resolves to the load balancer host and ingress that you supplied in the previous step. For example:
+
+   ```
+   http://${HOSTNAME}:${LB_PORT}/
+   ```
+   Where:
+
+     * `${HOSTNAME}` is where the ingress load balancer is running, in this case, the `External-IP` address of the Traefik load balancer.
```

Comment:
I do not think we need to say anything about External IP

Reply:
I'll remove that portion of the sentence.